docs/book/how-to/develop-locally/local-prod-pipeline-variants.md

```diff
@@ -0,0 +1,135 @@
+---
+description: Create different variants of your pipeline for local development and production.
+---
+
+# Create pipeline variants for local development and production
+
+When developing ZenML pipelines, it's often beneficial to have different variants of your pipeline for local development and production environments. This approach allows you to iterate quickly during development while maintaining a full-scale setup for production. While configuration files are one way to achieve this, you can also implement this directly in your code.
+
+## Creating pipeline variants
+
+There are several ways to create different variants of your pipeline:
+
+1. Using configuration files
+2. Implementing variants in code
+3. Using environment variables
+
+Let's explore each of these methods:
+
+### Using configuration files
```

Comment:
hmm this was a ### heading but then afterwards I expected the variants in code and env variables to be also ### headings ... maybe the structure needs work?